package.json

```diff
@@ -32,12 +32,13 @@
     "bundle-require": "^4.0.1",
     "esbuild": "^0.19.12",
     "eslint": "^9.16.0",
-    "glob": "^10.4.5",
+    "glob": "^11.0.1",
     "lighthouse": "^12.0.0",
     "lighthouse-logger": "2.0.1",
     "multi-progress-bars": "^5.0.3",
     "nx": "19.8.13",
     "parse-lcov": "^1.0.4",
+    "rimraf": "^6.0.1",
```

Comment:
Are we using `rimraf` directly? :raised_eyebrow:

Reply:
I don't think we are, the report refers to transitive dependency, on other hand I wanted to use it for test folders cleanup anyway 😀